libpldmresponder/fru.cpp

```diff
@@ -921,7 +921,9 @@ void FruImpl::processFruPresenceChange(const DbusChangedProps& chProperties,
 #ifdef OEM_IBM
     if (fruInterface == "xyz.openbmc_project.Inventory.Item.PCIeDevice")
     {
-        if (!pldm::responder::utils::checkIfIBMCableCard(fruObjPath))
+        bool isPresent =
+            pldm::responder::utils::checkFruPresence(fruObjPath.c_str());
+        if (!isPresent)
```

Comment:
can we change it to if(!pldm::responder::utils::checkFruPresence(fruObjPath.c_str())  to reduce the bool expression?

Reply:
ok how do we check for OEM_IBM?

Reply:
we do not need isPresent param right to check for OEM_IBM. we can skip creating that param is what I suggested.

Reply:
ok done